GliaWidgets/Sources/ViewModel/Chat/ChatViewModel.swift

```diff
@@ -168,18 +168,7 @@ class ChatViewModel: EngagementViewModel {
 
     override func start() {
         super.start()
-
-        loadHistory { [weak self] history in
-            guard let self = self else { return }
-            // We only proceed to considering enqueue flow if `startAction` is about starting of engagement.
-            guard case .startEngagement = self.startAction else { return }
-            // We enqueue eagerly in case if this is the first engagement for visitor (by  evaluating previous chat history)
-            // or in case if engagement has been restored.
-
-            if history.isEmpty || self.environment.getCurrentEngagement() != nil {
-                self.enqueue(mediaType: .text)
-            }
-        }
+        loadHistory()
```

Comment:
don't we need anymore to enqueue after history is fetched and current engagement exists?

Reply:
Enqueue happens only when visitor has allowed live observation